feature/visited/src/main/kotlin/uk/govuk/app/visited/data/VisitedItemsTransformer.kt

```diff
@@ -21,7 +21,7 @@ fun transformVisitedItems(visitedItems: List<VisitedItemUi>, todaysDate: LocalDa
             it.toLocalDate().year != todaysDate.year
     }
 
-    fun VisitedItemUi.previousMonthKey() = "${toLocalDate().month.name} ${toLocalDate().year}"
+    fun VisitedItemUi.previousMonthKey() = "${capitaliseMonth(toLocalDate().month.name)} ${toLocalDate().year}"
```

Comment:
Just a thought that capitaliseMonth() (which I'm aware hasn't changed as part of this PR) is specifically named where it seems it's use could be for any string? It could be a string extension function with a more generic name for reuse with not just months maybe?

Reply:
Yeah, it could be generic you're right. I'm happy to rename - perhaps when we need it for something other than months.